spec/support/examples/processor.rb

```diff
@@ -0,0 +1,80 @@
+      let(:integrity_attribute) { {"integrity" => "true"} }
+
+      it "should parse the stylesheet tag" do
+        expect(processor.stylesheet_link_tag(css_source)).to eql("<link rel=\"stylesheet\" media=\"screen\" href=\"#{processor.asset_path(css_source)}\"/>")
+      end
+
+      context "if css file is for print" do
+        it "should parse the stylesheet tag and extra options" do
+          expect(processor.stylesheet_link_tag(css_source, css_options)).to eql("<link rel=\"stylesheet\" media=\"print\" href=\"#{processor.asset_path(css_source)}\"/>")
+        end
+      end
+
+      context "if the integrity attribute is present, it should ignore it" do
+        it "should parse the stylesheet tag without the integrity attribute" do
+          expect(processor.stylesheet_link_tag(css_source, integrity_attribute)).to eql("<link rel=\"stylesheet\" media=\"screen\" href=\"#{processor.asset_path(css_source)}\"/>")
+        end
+      end
+    end
+
+    describe "#javascript_include_tag" do
+      let(:js_options)          { {"charset" => "UTF-8"} }
+      let(:integrity_attribute) { {"integrity" => "true"} }
+
+      it "should parse the javascript tag" do
+        expect(processor.javascript_include_tag(js_source)).to eql("<script src=\"#{processor.asset_path(js_source)}\"></script>")
+      end
+
+      it "should parse the javascript tag and extra options" do
+        expect(processor.javascript_include_tag(js_source, js_options)).to eql("<script src=\"#{processor.asset_path(js_source)}\" charset=\"UTF-8\"></script>")
+      end
+
+      it "if the integrity attribute is present, it should ignore it" do
+        expect(processor.javascript_include_tag(js_source, integrity_attribute)).to eql("<script src=\"#{processor.asset_path(js_source)}\"></script>")
+      end
+    end
+
+    describe "#content_tag" do
+      it "should return the correct html script tag" do
+        expect(processor.content_tag(:script, " src=\"#{processor.asset_path(js_source)}\"")).to eql("<script src=\"#{processor.asset_path(js_source)}\"></script>")
+      end
+    end
+
+    describe "#tag" do
+      it "should return the correct html link tag" do
+        expect(processor.tag(:link, " rel=\"stylesheet\" media=\"screen\" href=\"#{processor.asset_path(css_source)}\"")).to eql("<link rel=\"stylesheet\" media=\"screen\" href=\"#{processor.asset_path(css_source)}\"/>")
+      end
+    end
+
+    describe "#tag_options" do
+      let(:options) { {"rel"=>"stylesheet", "media"=>"screen", "href"=>processor.asset_path(css_source) } }
+
+      it "should parse the hash" do
+        expect(processor.tag_options(options)).to eql(" rel=\"stylesheet\" media=\"screen\" href=\"#{processor.asset_path(css_source)}\"")
+      end
+    end
+
+    describe "#exclude_sri_fields" do
+      let(:options) { {"rel"=>"stylesheet", "media"=>"screen", "href"=>processor.asset_path(css_source), "integrity" => true, "crossorigin" => "anonymous" } }
+
+      it "should parse the hash" do
```

Comment:
"should parse the hash" could be more descriptive.  "removes integrity and crossorigin keys from the hash" perhaps?